subsys/testsuite/ztest/include/ztest_mock.h

```diff
@@ -55,33 +55,33 @@
 /**
  * @brief Tell function @a func to expect the data @a data for @a param
  *
- * When using ztest_check_expected_data(), tell that the data ponted by @a param
- * should be same as data at @a data.
- * Only data pointer is stored by this function so it must still be valid when
- * ztest_check_expected_data is called.
+ * When using ztest_check_expected_data(), the data pointed to by
+ * @a param should be same @a data in this function. Only data pointer is stored
```

Comment:
I think these changes should be in a separate commit

Reply:
(I mean your bug fix should be one commit and all your clean-ups in another)

Reply:
Done